app/models/registerable_edition.rb

```diff
@@ -1,3 +1,9 @@
+# RegisterableEdition is an object that `GdsApi::Panopticon::Registerer` uses
+# to generate a payload to send to panopticon.
+#
+# The reason we're still sending things to panopticon is to make them available
+# to link to as "related artefacts" and to make the need_ids available for
```

Comment:
Is there any external documentation you can link to here that would explain what related artefacts, need ids, etc are and what uses them?